### ISSUE TITLE
How about Raspberry Pi's Webcam?

### ISSUE THREAD

**Issue by berkaysit** [rijinmk/code-qr-code-reader-rpi3#1]
How can we use without any USB webcam but Raspverry Pi's original webcam?

**Comment by samrudh** [comment]
yes you can use the Raspberry Pi's original webcam. In that case you might want to use your own function to capture image in reader.py. 
My answer [here](https://github.com/rijinmk/code-qr-code-reader-rpi3/issues/2) or my code [here](https://github.com/samrudh/code-qr-code-reader-rpi3) might help you
